exporter/opentelemetry-exporter-otlp/pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "opentelemetry-exporter-otlp"
+dynamic = ["version"]
+description = "OpenTelemetry Collector Exporters"
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.7"
+authors = [
+  { name = "OpenTelemetry Authors", email = "cncf-opentelemetry-contributors@lists.cncf.io" },
+]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Typing :: Typed",
+]
+dependencies = [
+  "opentelemetry-exporter-otlp-proto-grpc == 1.12.0",
+  "opentelemetry-exporter-otlp-proto-http == 1.12.0",
+]
+
+[project.entry-points.opentelemetry_logs_exporter]
+otlp = "opentelemetry.exporter.otlp.proto.grpc._log_exporter:OTLPLogExporter"
```

Comment:
@srikanthccv @ocelotl 
Do we want the entry points to be defined in the generic package?

Reply:
Correct, the protocol + encoding specific exporter has a different entry point name such `otlp_proto_http`. This convenience package just defaults to grpc.